src/Services/Template.php

```diff
@@ -37,6 +38,13 @@ public function timeout(): int
             : (int) Config::get('enso.imports.timeout');
     }
 
+    public function type(): string
+    {
+        return $this->template->has('type')
+            ? $this->template->get('type')
+            : ImportType::Insert;
+    }
```

Comment:
we should also validate the type in template validation

Reply:
we don't need to type anymore